domains/future.is-an.app.js

```diff
@@ -0,0 +1,12 @@
+addSubDomain({
+  description: 'domain for my website',
+  domain: 'is-an.app',
+  subdomain: 'future',
+  owner: {
+    email: 'gslicocuk8080@yaani.com',
+  },
+  record: {
+    CNAME: 'github.com/The-Future-Software-Company',
```

Comment:
Wrong cname record

Reply:
What is wrong?

Reply:
https://en.wikipedia.org/wiki/List_of_DNS_record_types

Reply:
https://github.com/tarampampam/free-domains/pull/134#discussion_r1118403016